spec/tasks/gen/resource_spec.cr

```diff
@@ -17,6 +17,11 @@ describe Gen::Action do
           "./src/actions/users/edit.cr": "class Users::Edit < BrowserAction",
           "./src/actions/users/update.cr": "class Users::Update < BrowserAction",
           "./src/actions/users/delete.cr": "class Users::Delete < BrowserAction"
+        should_create_files_with_contents io,
```

Comment:
I think this is a great idea. I think for this PR it would be better to keep it as-is so we can merge it in, but if you'd like to make a separate PR adding some fixture files to test against, that'd be great! No pressure though.